pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -216,3 +220,34 @@ func findContainerIDByName(status *v1.PodStatus, name string) (string, error) {
 	}
 	return "", fmt.Errorf("[cpumanager] unable to find ID for container with name %v in pod status (it may not be running)", name)
 }
+
+// Admit rejects a pod if its not safe to admit for node stability.
```

Comment:
Please add a note here that this function must be part of `*manager`'s method set in order to conform to the `lifecycle.PodAdmitHandler` interface.